schema_salad/python_codegen_support.py

```diff
@@ -84,6 +91,36 @@ def __init__(
                 self.vocab[k] = v
                 self.rvocab[v] = k
 
+    @property
+    def graph(self) -> Graph:
```

Comment:
Needs a docstring. Maybe a more descriptive name? How about caching the result based upon the hash of the schemas?

Reply:
For the docstring, what about
``python
"""
    Generate rdflib.Graph objects from loaded schemas.
"""
```

Fot the name, I named it `graph` because in `cwltool` these objects are stored in the `graph` attribute.

For the cache key, you mean something like
```python
for schema in self.schemas:
    fetchurl = self.fetcher.urljoin(self.fileuri, schema)
    content = self.fetcher.fetch_text(fetchurl)
    key = hash(content)
    if key not in self.cache or self.cache[key] is True:
        ...
```

PS (I am still ignoring the meaning of `if self.cache[key] is True`. Why should it check something like that?)

Reply:
```suggestion
    def graph(self) -> Graph:
    """Generate a merged rdflib.Graph from all entries in self.schemas."""
```

For the cache key: check even before this loop, if the schemas haven't changed and we already downloaded them, made graphs, and merged those graphs, then why do it again?
``` python
key = hash(self.schemas)
if key in self.cache:
  return cast(Graph, self.cache[key])
# and continue with line 102 onwards
# and later
self.cache[key] = graph
```

Reply:
Added docstring. For the `hash` part, I have to construct a `tuple` because lists are not hashable.